cockroachcloud/quickstart.md

```diff
@@ -6,429 +6,119 @@ referral_id: docs_quickstart_free
-<section class="filter-content" markdown="1" data-scope="mac linux">
-In a terminal set the `JDBC_DATABASE_URL` environment variable to the JDBC connection string:
-
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-export JDBC_DATABASE_URL="<jdbc-connection-string>"
-~~~
-
-The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
-</section>
-{% comment %} End mac linux {% endcomment %}
-
 <section class="filter-content" markdown="1" data-scope="windows">
-In a terminal set the `JDBC_DATABASE_URL` environment variable to the JDBC connection string:
+In a PowerShell window, enter the following command to download and extract the `ccloud` binary and add it to your `PATH`:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$env:JDBC_DATABASE_URL = "<jdbc-connection-string>"
+$ErrorActionPreference = "Stop"; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; $ProgressPreference = 'SilentlyContinue'; $null = New-Item -Type Directory -Force $env:appdata/ccloud; Invoke-WebRequest -Uri https://binaries.cockroachdb.com/ccloud/ccloud_windows-amd64_{{ ccloud_version }}.zip -OutFile ccloud.zip; Expand-Archive -Force -Path ccloud.zip; Copy-Item -Force ccloud/ccloud.exe -Destination $env:appdata/ccloud; $Env:PATH += ";$env:appdata/ccloud"; # We recommend adding ";$env:appdata/ccloud" to the Path variable for your system environment. See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables#saving-changes-to-environment-variables for more information.
 ~~~
-
-The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
 </section>
-{% comment %} End windows {% endcomment %}
 
-{{site.data.alerts.callout_success}}
-For reference information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../../docs/stable/connect-to-the-database.html).
-{{site.data.alerts.end}}
+## Run `ccloud quickstart`
 
-</section>
-{% comment %} End java {% endcomment %}
+The `ccloud quickstart` command guides you through logging in to CockroachDB Cloud, creating a new {{ site.data.products.serverless }} with a $0 spend limit, and connecting to the new cluster. 
 
-## Run the sample code
-
-<section class="filter-content" markdown="1" data-scope="node">
-
-1. Clone the `quickstart-code-samples` repo:
+1. In a terminal, run `ccloud quickstart`:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    git clone https://github.com/cockroachdb/quickstart-code-samples
+    ccloud quickstart
     ~~~
 
-1. Navigate to the `node` directory of the repo:
+1. Log in to {{ site.data.products.db }} using the browser window opened by `ccloud quickstart`. If you are new to {{ site.data.products.db }}, you can register using one of the single sign-on (SSO) options, or create a new account using an email address.
```

Comment:
Style nit (applies to following steps too): Since the second sentence is not required to do the task, start the second sentence ("If you are new...") on the next line. This helps with readability.

Similarly, the next step should have "Each prompt..." start on the next line.